configs/sites/tier1/orion/packages_intel.yaml

```diff
@@ -30,4 +30,4 @@ packages:
     - '@1.2.1 ~mkl'
   py-numpy:
     require::
-    - '@:1.23 ^openblas'
+    - '^openblas'
```

Comment:
If you want to be shmancy you could use an edge label here: `^[virtuals=blas,lapack] openblas` but functionally I don't think it makes a difference.

Is there a reason to have this specified redundantly with the general blas/lapack provider settings?

Reply:
This gets back to the openblas vs mkl debate. With Intel, the default provider is MKL (specificied in common/packages_intel.yaml). You said back then that you'd prefer the NOAA HPCs to keep using openblas until you have a chance of testing with MKL. Therefore, all the NOAA HPCs overwrite the virtual provider for blas/lapack with openblas. I found during concretizing that spack sometimes ignored this for numpy and used MKL directly (maybe because the virtual provider wasn't configured correctly in the package and/or our setup?).

Reply:
Got it. I'm fine with this as is, and if we find a neater way to do the logic later then cool.